package.json

```diff
@@ -54,6 +54,7 @@
     "@types/cookie-parser": "^1.4.2",
     "@types/csurf": "^1.11.2",
     "@types/express": "^4.17.13",
+    "@types/http-errors": "^1.8.1",
```

Comment:
you've added the types for this package, but i don't see the dependency itself, especially as I see the dependency being used later on

Reply:
i assume it's being pulled in by another dependency, but since we're using it now best to add it to our dependencies too

Reply:
ah, great point! It was added so we can use types for the `createError` method imported from `http-errors`. I'll add it in as an explicit dep!